tests/stub/summary/test_summary.py

```diff
@@ -247,6 +235,793 @@ def test_multiple_notifications(self):
+                summary.gql_status_objects[0]
+            )
+            self.assert_is_test_notification_as_gql_status_object(
+                summary.gql_status_objects[1]
+            )
+
+    def test_warning_no_data(self):
+        in_notifications = [self.make_test_notification()]
+        summary = self._get_summary(
+            "summary_with_notifications_no_data.script",
+            vars_={
+                "#NOTIFICATIONS#": json.dumps(in_notifications)
+            }
+        )
+        self.assertEqual(len(summary.gql_status_objects), 2)
+        if self._server.count_requests("PULL"):
+            self.assert_is_no_data(summary.gql_status_objects[0])
+        else:
+            # For drivers that lazily PULL records.
+            # They cannot know whether there is no data until the user
+            # makes the driver try to pull records.
+            self.assert_is_no_data_unknown_subclass(
+                summary.gql_status_objects[0]
+            )
+        self.assert_is_test_notification_as_gql_status_object(
+            summary.gql_status_objects[1]
+        )
+
+    def test_information(self):
+        in_notifications = [
+            self.make_test_notification(severity="INFORMATION")
+        ]
+        summary = self._get_summary(
+            "summary_with_notifications.script",
+            vars_={
+                "#NOTIFICATIONS#": json.dumps(in_notifications)
+            }
+        )
+        self.assertEqual(len(summary.gql_status_objects), 2)
+        if self._server.count_requests("PULL"):
+            self.assert_is_success(summary.gql_status_objects[0])
+        else:
+            # For drivers that lazily PULL records.
+            # They cannot know whether there is no data until the user
+            # makes the driver try to pull records.
+            self.assert_is_no_data_unknown_subclass(
+                summary.gql_status_objects[0]
+            )
+        self.assert_is_test_notification_as_gql_status_object(
+            summary.gql_status_objects[1],
+            severity="INFORMATION", parsed_severity="INFORMATION"
+        )
+
+    def test_information_omitted_result(self):
+        in_notifications = [
+            self.make_test_notification(severity="INFORMATION")
+        ]
+        summary = self._get_summary(
+            "summary_with_notifications_omitted_result.script",
+            vars_={
+                "#NOTIFICATIONS#": json.dumps(in_notifications)
+            }
+        )
+        self.assertEqual(len(summary.gql_status_objects), 2)
+        if self._server.count_requests("PULL"):
+            self.assert_is_omitted_result(summary.gql_status_objects[0])
+        else:
+            # For drivers that lazily PULL records.
+            # They cannot know whether there is no data until the user
+            # makes the driver try to pull records.
+            self.assert_is_no_data_unknown_subclass(
+                summary.gql_status_objects[0]
+            )
```

Comment:
```suggestion
        self.assert_is_omitted_result(summary.gql_status_objects[0])
```
https://github.com/neo-technology/drivers-adr/pull/89/files#diff-2c370520cd34650b73ee220b6414ba2ed657d6ee495849ac3f20eee2112439f7R316

If I understand the above right, the `else` block should still have the omitted result because no records are returned by the server and the keys are empty.